test/cpp/microbenchmarks/helpers.cc

```diff
@@ -98,10 +94,6 @@ void TrackCounters::AddToLabel(std::ostream& out, benchmark::State& state) {
       << " nows/iter:"
       << ((double)(gpr_atm_no_barrier_load(&gpr_now_call_count) -
                    now_calls_at_start_) /
-          (double)state.iterations())
-      << " allocs/iter:"
```

Comment:
This was actually a fairly useful thing to track.  Is there some other way we can track this?

Reply:
There are various [alternatives](https://stackoverflow.com/questions/1696671/how-detect-programatically-count-of-bytes-allocated-by-process-on-heap) to this but none of them is easy to use in unit-tests. I'll research this more later on.

Reply:
Okay.  I'd probably be more insistent about this if we were still seeing this output directly in each PR, but ever since this was moved to another tab, no one looks at it anyway. :(

I would like to see this fixed, but for now, let's just add a TODO.

Reply:
I updated this to enable memory counter without a custom allocator using "--wrap=symbol" from ld. This can work only with gcc ld but I think it's enough.